cairo/src/helper.rs

```diff
@@ -43,22 +43,14 @@ impl<F: Field> CairoFieldHelpers<F> for F {
         acc
     }
 
-    fn to_hex_le(self) -> String {
+    fn to_hex_be(self) -> String {
         let mut bytes = self.to_bytes();
         bytes.reverse();
         hex::encode(bytes)
     }
 
     fn vec_to_field(vec: &[i128]) -> Vec<F> {
-        vec.iter()
-            .map(|i| {
-                if *i < 0 {
-                    -F::from((-(*i)) as u64)
-                } else {
-                    F::from((*i) as u64)
-                }
```

Comment:
note that this function exists here: https://github.com/o1-labs/proof-systems/blob/master/utils/src/field_helpers.rs#L90

Reply:
You're right, good catch, I fixed this in the [i128 field helper](https://github.com/o1-labs/proof-systems/pull/493/commits/7dca96d7b3de965450947141c41ff95412def4e8) commit. Actually, I suggested that in that field_helper we could actually perform the casting using `u32` instead of `u64` to convert `i32` to fields.